contracts/scripts/update-round.ts

```diff
@@ -6,7 +6,7 @@ import params from './config/update-round.config';
 
 const { ethers } = hre;
 
-const ipfs = createIpfs(process.env.FLEEK_STORAGE_API_KEY!);
+const ipfs = createIpfs(process.env.FLEEK_STORAGE_API_KEY!); // eslint-disable-line @typescript-eslint/no-non-null-assertion
```

Comment:
This was choking when I ran `yarn lint` from root - not required, but nice to have 🤷

Reply:
This works because we polyfill `process.env`, but the Vite way to do this would be 
```suggestion
const ipfs = createIpfs(import.meta.env.FLEEK_STORAGE_API_KEY);
```

This is how we import env vars elsewhere in the app, and I think would remove the need for this TS change. https://vitejs.dev/guide/env-and-mode.html

Reply:
Oh wait nevermind we're in the `contracts` dir here, sorry!